seqr/views/apis/staff_api_tests.py

```diff
@@ -0,0 +1,173 @@
+          "genomeVersion_1": "38",
+          "sampleType": "WGS",
+          "sourceFilePath": TEST_INDEX_SECOND_SRC_PATH,
+          "hail_version_1": "0.2.24",
+          "gencodeVersion_1": "25"
+        },
+        "_all": {
+          "enabled": False
+        },
+      }
+    }
+  },
+  "test_index_old": {
+    "mappings": {
+      "variant": {
+        "_meta": {
+          "gencodeVersion": "25",
+          "hail_version": "0.2.24",
+          "genomeVersion": "38",
+          "sampleType_1": "WGS",
+          "sourceFilePath_1": "gs://seqr-datasets/v02/GRCh38/RDG_WGS_Broad_Internal/v10/sharded_vcf/RDG_Broad_WGS_Internal.filtered.*.vcf.gz",
+          "genomeVersion_1": "38",
+          "sampleType": "WGS",
+          "sourceFilePath": TEST_INDEX_SRC_PATH,
+          "hail_version_1": "0.2.24",
+          "gencodeVersion_1": "25"
+        },
+        "_all": {
+          "enabled": False
+        },
+      }
+    }
+  },
+  "1kg.vcf.gz": {
+    "mappings": {
+      "variant": {
+        "_meta": {
+          "gencodeVersion": "19",
+          "genomeVersion": "37",
+          "sampleType": "WES",
+          "datasetType": "VARIANTS",
+          "sourceFilePath": TEST_INDEX_1KG_SRC_PATH
+        },
+        "_all": {
+          "enabled": False
+        },
+      }
+    }
+  }
+}
+from django.test import TestCase
+from django.urls.base import reverse
+
+from seqr.views.apis.staff_api import elasticsearch_status
+from seqr.views.utils.test_utils import _check_login
+
+class VariantSearchAPITest(TestCase):
+    fixtures = ['users', '1kg_project']
+    multi_db = True
+
+    @mock.patch('elasticsearch_dsl.index.Index.get_mapping')
+    @mock.patch('elasticsearch.client.cat.CatClient.aliases')
+    @mock.patch('elasticsearch.client.cat.CatClient.indices')
+    @mock.patch('elasticsearch.client.cat.CatClient.allocation')
+    def test_elasticsearch_status(self, mock_cat_allocation, mock_cat_indices, mock_cat_aliases, mock_get_mapping):
+        url = reverse(elasticsearch_status)
+        _check_login(self, url)
+
+        mock_cat_allocation.return_value = ES_CAT_ALLOCATION
+        mock_cat_indices.return_value = ES_CAT_INDICES
+        mock_cat_aliases.return_value = ES_CAT_ALIAS
+        mock_get_mapping.return_value = ES_INDEX_MAPPING
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 200)
+        response_json = response.json()
+        self.assertListEqual(response_json.keys(), ['indices', 'errors', 'diskStats', 'elasticsearchHost'])
+        for index in response_json['indices']:
+            index_id = index['index']
+            mapping = ES_INDEX_MAPPING[index_id]
+            self.assertEqual(index['sourceFilePath'], mapping['mappings']['variant']['_meta']['sourceFilePath'])
+        node_list = []
+        for node in response_json['diskStats']:
+            node_list.append(node['node'])
```

Comment:
This would be much more readable as a list comprehension: `node_list = [node['node'] for node in response_json['diskStats'] ]`

Reply:
Updated.